docs/tutorials/javascript/build-apps/credential-issuing-service.md

```diff
@@ -0,0 +1,261 @@
+---
+seo:
+    description: Build a credential issuing microservice with Javascript and Node.js.
+---
+
+# Build a Credential Issuing Service
+
+_(Requires the Credentials amendment. {% not-enabled /%})_
+
+This tutorial demonstrates how to build and use a microservice that issues [Credentials](../../../concepts/decentralized-storage/credentials.md) on the XRP Ledger, in the form of a RESTlike API, using the [Express](https://expressjs.com/) framework for Node.js.
```

Comment:
Suggestion: add a "See also: the Python version of this tutorial" link. (And the reciprocal link from there.) That way, if someone finds one page via a search but prefers the other programming language, they have a direct path. (I think it might also help SEO and AI understand the connection between the pages.)